### DISCUSSION TITLE
Webmention.io domain call empty

### DISCUSSION THREAD

**Discussion by kfitz** [snarfed/bridgy#1512] [Q&A] [answered]
Hi! I am trying to run down the source of a problem I'm having somewhere between Bridgy, Webmention,io, and 11ty, and coming up empty. I'm hoping you might have an idea what's going on.

I'm using a lightly modified version of Max Böck's [eleventy-webmentions](https://github.com/maxboeck/eleventy-webmentions) starter on an 11ty site at [kfitz.info](https://kfitz.info/). I am building locally and then pushing the resulting site to my server. I definitely have webmentions in my dashboard that should be available for fetching, and nothing is throwing any errors, but no mentions are retrieved when I build. This is the key bit of the API request in webmentions.js:

```
let url = ${API_ORIGIN}?domain=${domain}&token=${TOKEN}
if (since) {
url += &per-page=100&&since=${since}
} else {
url += &per-page=999
}

const response = await fetch(url)
if (response.ok) {
const feed = await response.json()
console.log(
${feed.children.length} webmentions fetched from ${API_ORIGIN}
)
return feed
}

```
API_ORIGIN is set to https://webmention.io/api/mentions.jf2, my domain is set correctly, and my token is as well. So the api url that’s supposed to pull all the mentions for this domain and then let them be processed by page resolves like this:

`http://webmention.io/api/mentions.jf2?domain=kfitz.info&token=${TOKEN}`

However, when I run it it comes up empty. Zero. Nothing. However, a page-specific call like this contains all the mentions I'd expect:

`https://webmention.io/api/mentions.jf2?target=https://kfitz.info/eleventy/`

I've confirmed that my token is entered correctly, and I've confirmed that webmention.io is not returning any errors. It's just giving me zero results. So I'm wondering if it's possible that what Bridgy is feeding into webmention.io is somehow missing the domain despite pinging the specific pages correctly?

**Comment by snarfed** [comment, marked as answer]
Hmm! Sorry for the trouble! Looking at the first webmention in https://webmention.io/api/mentions.jf2?target=https://kfitz.info/eleventy/ and its Bridgy source page https://brid.gy/comment/mastodon/@kfitz@hcommons.social/110697247235385780/110697528142328000 , both correctly have https://kfitz.info/eleventy/ as the in-reply-to, and webmention.io correctly has it as wm-target. So, I suspect this is on webmention.io's side. https://github.com/aaronpk/webmention.io/issues/188 is probably the right place to follow up.

**Reply by kfitz** [reply]
Thank you! I'm glad to at least be localizing the issue :)

**Reply by snarfed** [reply]
We think https://github.com/aaronpk/XRay/pull/120 probably fixed this. Feel free to retry!
